plugins/uploadimage/plugin.js

```diff
@@ -146,5 +152,11 @@
 	 * @since 4.5
 	 * @cfg {String} [imageUploadUrl='' (empty string = disabled)]
 	 * @member CKEDITOR.config
+	 *
+	 * Since 4.8.1 version this setting allows to configure custom types for image upload. If not provided default types will be used (jpeg, png, gif, bmp).
+	 * Warning: some browsers does not support custom types eg. Google Chrome and Firefox cannot display `TIF` images.
```

Comment:
This information is not precise. Chrome and Firefox do support custom types, but not all of them.